packages/go_router/lib/src/router.dart

```diff
@@ -26,16 +23,19 @@ class GoRouter extends ChangeNotifier with NavigatorObserver {
   /// and an error page builder.
   GoRouter({
     required List<GoRoute> routes,
+    // TODO(johnpryan): Change to a route, improve error API
     GoRouterPageBuilder? errorPageBuilder,
     GoRouterWidgetBuilder? errorBuilder,
     GoRouterRedirect? redirect,
     Listenable? refreshListenable,
     int redirectLimit = 5,
     bool routerNeglect = false,
     String? initialLocation,
+    // TODO(johnpryan): Deprecate this parameter
```

Comment:
I can imagine people would want to control their path strategy, are we going to replace it with a different api?

Reply:
The `flutter_web_plugins` package (part of the SDK) now takes care of the conditional imports for you, so using this API is simpler:

```dart
import 'package:flutter/material.dart';
import 'package:flutter_web_plugins/url_strategy.dart';

void main() {
  usePathUrlStrategy();
  runApp(ExampleApp());
}
```

See also: https://github.com/flutter/website/issues/7386

Reply:
sounds good, can you add the issue link to the todo? same thing as other TODO that may have issue associate with it

Reply:
Filed an issue: https://github.com/flutter/flutter/issues/108132

Reply:
can you link this issue in the todo here?

Reply:
here and everywhere

Reply:
Done